vision_ocv/CMakeLists.txt

```diff
@@ -24,6 +23,7 @@ set(TEST_DATA ${CMAKE_SOURCE_DIR}/${PROJ_MAIN_NAME}/tests/data)
 # ADD SUB FOLDERS
 # ##############################################################################
 add_subdirectory(modules)
+add_subdirectory(benchmark)
```

Comment:
Wrap this in a `if (KPSR_TEST_PERFORMANCE)` so that the benchmarks are optional.